### ISSUE TITLE
 Error: Uncaught (in promise): TypeError: Cannot read property 'updated_at' of undefinedTypeError: Cannot read property 'updated_at' of undefined

### ISSUE THREAD

**Issue by hahla** [johannesjo/super-productivity#1624]
### Steps to Reproduce

1. I put my macbook to sleep, I guess maybe the app was still running some timer?
2. the app was running on a secondary display connected via usb-c
3.
4.

### Error Log (Desktop only)
Error: Uncaught (in promise): TypeError: Cannot read property 'updated_at' of undefined TypeError: Cannot read property 'updated_at' of undefined at t. (file:///Applications/Super%20Productivity.app/Contents/Resources/app.asar/dist/main.1a6b823be2f8c4954b23.js:1:2544769) at Generator.next () at o (file:///Applications/Super%20Productivity.app/Contents/Resources/app.asar/dist/main.1a6b823be2f8c4954b23.js:1:2542438) at l.invoke (file:///Applications/Super%20Productivity.app/Contents/Resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:6534) at Object.onInvoke (file:///Applications/Super%20Productivity.app/Contents/Resources/app.asar/dist/main.1a6b823be2f8c4954b23.js:1:140387) at l.invoke (file:///Applications/Super%20Productivity.app/Contents/Resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:6474) at t.run (file:///Applications/Super%20Productivity.app/Contents/Resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:1933) at file:///Applications/Super%20Productivity.app/Contents/Resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:16742 at l.invokeTask (file:///Applications/Super%20Productivity.app/Contents/Resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:7152) at Object.onInvokeTask (file:///Applications/Super%20Productivity.app/Contents/Resources/app.asar/dist/main.1a6b823be2f8c4954b23.js:1:140203)
this._gitlabApiService.getByIds$ (webpack:///src/app/features/issue/providers/gitlab/gitlab-common-interfaces.service.ts:161:53)toString (file:///Applications/Super%20Productivity.app/Contents/Resources/app.asar/dist/main.1a6b823be2f8c4954b23.js:1:2542438)invoke (webpack:///node_modules/zone.js/fesm2015/zone.js:372:25)onInvoke (webpack:///node_modules/@angular/core/fesm2015/core.js:28672:32)invoke (webpack:///node_modules/zone.js/fesm2015/zone.js:371:51)run (webpack:///node_modules/zone.js/fesm2015/zone.js:134:42)zone.scheduleMicroTask(source, (webpack:///node_modules/zone.js/fesm2015/zone.js:1276:35)invokeTask (webpack:///node_modules/zone.js/fesm2015/zone.js:406:30)onInvokeTask (webpack:///node_modules/@angular/core/fesm2015/core.js:28659:32)scheduleResolveOrReject (webpack:///node_modules/zone.js/fesm2015/zone.js:1213:30)makeResolver (webpack:///node_modules/zone.js/fesm2015/zone.js:1120:16)toString (file:///Applications/Super%20Productivity.app/Contents/Resources/app.asar/dist/main.1a6b823be2f8c4954b23.js:1:2561001)invoke (webpack:///node_modules/zone.js/fesm2015/zone.js:372:25)onInvoke (webpack:///node_modules/@angular/core/fesm2015/core.js:28672:32)invoke (webpack:///node_modules/zone.js/fesm2015/zone.js:371:51)run (webpack:///node_modules/zone.js/fesm2015/zone.js:134:42)zone.scheduleMicroTask(source, (webpack:///node_modules/zone.js/fesm2015/zone.js:1276:35)invokeTask (webpack:///node_modules/zone.js/fesm2015/zone.js:406:30)onInvokeTask (webpack:///node_modules/@angular/core/fesm2015/core.js:28659:32)
META: SP7.6.0 Electron – en-US – MacIntel – Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) superProductivity/7.6.0 Chrome/91.0.4472.124 Electron/13.1.7 Safari/537.36

Usually, you can find it here:
on Linux: ~/.config/superProductivity/logs/main.log
on macOS: ~/Library/Logs/superProductivity/main.log
on Windows: %USERPROFILE%/AppData/Roaming/superProductivity/logs/main.log
. -->

### Stacktrace
```
this._gitlabApiService.getByIds$ (webpack:///src/app/features/issue/providers/gitlab/gitlab-common-interfaces.service.ts:161:53)
toString (file:///Applications/Super%20Productivity.app/Contents/Resources/app.asar/dist/main.1a6b823be2f8c4954b23.js:1:2542438)
invoke (webpack:///node_modules/zone.js/fesm2015/zone.js:372:25)
onInvoke (webpack:///node_modules/@angular/core/fesm2015/core.js:28672:32)
invoke (webpack:///node_modules/zone.js/fesm2015/zone.js:371:51)
run (webpack:///node_modules/zone.js/fesm2015/zone.js:134:42)
zone.scheduleMicroTask(source, (webpack:///node_modules/zone.js/fesm2015/zone.js:1276:35)
invokeTask (webpack:///node_modules/zone.js/fesm2015/zone.js:406:30)
onInvokeTask (webpack:///node_modules/@angular/core/fesm2015/core.js:28659:32)
scheduleResolveOrReject (webpack:///node_modules/zone.js/fesm2015/zone.js:1213:30)
makeResolver (webpack:///node_modules/zone.js/fesm2015/zone.js:1120:16)
toString (file:///Applications/Super%20Productivity.app/Contents/Resources/app.asar/dist/main.1a6b823be2f8c4954b23.js:1:2561001)
invoke (webpack:///node_modules/zone.js/fesm2015/zone.js:372:25)
onInvoke (webpack:///node_modules/@angular/core/fesm2015/core.js:28672:32)
invoke (webpack:///node_modules/zone.js/fesm2015/zone.js:371:51)
run (webpack:///node_modules/zone.js/fesm2015/zone.js:134:42)
zone.scheduleMicroTask(source, (webpack:///node_modules/zone.js/fesm2015/zone.js:1276:35)
invokeTask (webpack:///node_modules/zone.js/fesm2015/zone.js:406:30)
onInvokeTask (webpack:///node_modules/@angular/core/fesm2015/core.js:28659:32)
```

### Meta Info
META: SP7.6.0 Electron – en-US – MacIntel – Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) superProductivity/7.6.0 Chrome/91.0.4472.124 Electron/13.1.7 Safari/537.36

### Actions Before Error
```
1635264386348: [WorkContext] Set Active Work Context 
1635264386354: [Task] SetSelectedTask 
1635264386390: [Note] Load Note State 
1635264386392: [Bookmark] Load Bookmark State 
1635264386393: [Project] Load Project related Data Success 
1635264386393: [Improvement] Clear Hidden Improvements 
1635264386393: [Improvement] Delete multiple Improvements 
1635264387983: [Layout] Show AddTaskBar 
1635264394553: [Task][Issue] Add Task 
1635264394560: [Persistence] Save to DB (2) 
1635264397852: [Layout] Hide AddTaskBar 
1635264407943: [Task] Update Task 
1635264407944: [Persistence] Save to DB 
1635264411393: [Layout] Show AddTaskBar 
1635264418667: [Task][Issue] Add Task 
1635264418674: [Persistence] Save to DB (2) 
1635264490982: [Idle] Reset (11) 
1635264496688: [Layout] Hide AddTaskBar 
1635266062556: [Idle] Reset (85) 
1635266072620: [WorkContext] Set Active Work Context 
1635266072628: [Task] SetSelectedTask 
1635266072654: [Note] Load Note State 
1635266072656: [Bookmark] Load Bookmark State 
1635266072656: [Project] Load Project related Data Success 
1635266072657: [Improvement] Clear Hidden Improvements 
1635266072657: [Improvement] Delete multiple Improvements 
1635266081716: [Task][Issue] Add Task 
1635266081722: [Persistence] Save to DB (2) 
1635318660433: [Idle] Reset (2379)
```

**Comment by ishanarora** [comment]
Hi,

I have been getting the exact same error stack for the past few days. The Network tab shows 4 requests to the gitlab api (one to get the issues, and three to get the notes for each of the 3 issues). All the gitlab responses are successful, and valid json. Unfortunately I can't share the actual network traffic for confidentiality reasons. The gitlab instance is self-hosted (version 13.12.1). Thanks.

**Comment by ishanarora** [comment]
Duplicate of #1634

**Comment by ishanarora** [comment]
Fixed in 48e2b919f
